src/debug/debug.fork.test.ts

```diff
@@ -60,3 +71,71 @@ test('works in forked scope', async () => {
     ]
   `);
 });
+
+test('trace support', async () => {
+  const buttonClicked = createEvent();
+  const inputChanged = createEvent();
+  const $form = createStore(0).on(inputChanged, (s) => s + 1);
+  const $formValid = $form.map((s) => s > 0);
+  const submitFx = createEffect(() => {});
+
+  $form.on(submitFx.doneData, (s) => s + 1);
+
+  sample({
+    source: $form,
+    clock: buttonClicked,
+    filter: $formValid,
+    target: submitFx,
+  });
+
+  debug({ trace: true }, $form, submitFx);
```

Comment:
What about to add `debug.trace($form, submitFx)` ?

Reply:
I think, this would be semantically different, like if traces are some alternative form of `debug` logs - but this is not the case, since traces are complementary to standard logs of `debug` 🤔 

Also there maybe need for some other configurations in the future, so i think that config is needed anyway